### ISSUE TITLE
Update aws_ssm_maintenance_window to support start/end dates

### ISSUE THREAD

**Issue by e-moshaya** [terraform-providers/terraform-provider-aws#6980]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Add support for two new optional arguments.

### New or Affected Resource(s)

aws_ssm_maintenance_window

### Potential Terraform Configuration

```hcl
resource "aws_ssm_maintenance_window" "maintenance_window" {
  name     = "My Maintenance Window"
  schedule = "cron(0 23 ? * *)"
  duration = "2"
  cutoff   = "0"
  start_date = "2018-10-01T10:10:10Z"
  end_date = "2018-11-01T10:10:10Z"
}
```

### References

* https://docs.aws.amazon.com/cli/latest/reference/ssm/create-maintenance-window.html
* https://docs.amazonaws.cn/en_us/systems-manager/latest/userguide/maintenance-windows-cli-tutorials-update.html

**Comment by bflad** [comment]
Pull request submitted to add `end_date`, `schedule_timezone`, and `start_date` attributes: #7040

**Comment by bflad** [comment]
Support for the new attributes mentioned above has been merged and will release with version 1.55.0 of the Terraform AWS provider, likely middle of this week.

**Comment by bflad** [comment]
This has been released in [version 1.55.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1550-january-10-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
